### ISSUE TITLE
When enabling Limit by subfolder no sub folder options appear anywhere

### ISSUE THREAD

**Issue by BAM-Dennis** [nystudio107/craft3-imageoptimize#379]
### Describe the bug
Documentation says there should be a place, where i could set that images in a subfolder only should be transformed.
i cannot find the option at all. Could this be related to the changes they have made here:
https://craftcms.com/blog/rethinking-volume-subfolders

### To reproduce
Steps to reproduce the behaviour:
1. Goto Settings and enable: "Limit by Sub-Folder"
2. Nothing happens at alle anywhere :)

### Expected behaviour
i expect subfolder settings to appear somewhere

### Screenshots
--

### Versions
- Plugin version: 4.0.4
- Craft version: 4.4.9

**Comment by khalwat** [comment]
Yeah I think with Craft redoing this, it may not be a concept that applies anymore.
